### ISSUE TITLE
Resource grouping in Access Requests or similar QOL improvement

### ISSUE THREAD

**Issue by elijahsgh** [gravitational/teleport#22163]
What would you like Teleport to do?
When creating an access request a user can specify globbed resources, etc. But when approving the access request it is often difficult to tell what resources or query was used.

The customer would like a way to group resources or have a more obvious output in the access request. Currently the resources appear as UUIDs, etc, and it can be cumbersome to translate them back to resources to ensure the user is requesting access to the correct and specific resources they should get access to.

What problem does this solve?
Quality-of-life improvement for multi-tenant environments.

If a workaround exists, please include it.

**Comment by zmb3** [comment]
Have a screenshot you could share?

I'm wondering if it's just https://goteleport.com/docs/access-controls/faq/#why-do-i-see-a-uuid-instead-of-a-hostname-when-reviewing-access-requests that you're seeing.

**Comment by elijahsgh** [comment]
Looks good. I'll follow up with the customer - I believe they were actually using tctl. Let me check in with them.

**Comment by skorobogatydmitry** [comment]
Yes, that's fair. It can be problematic for e.g. 100 resources, but that's not our case, I think.
Also, we have slack integration. Am I right that labels won't be shown in the slack notification nor list of resources ?

As for request side, we use `tsh` to request access, so unified resource view in UI doesn't help.
